### DISCUSSION TITLE
Never find the documentation for multiple authentication

### DISCUSSION THREAD

**Discussion by davidjpella** [inertiajs/inertia#474] [Help]
Hello Team, maybe I am missing something along the way but I find myself really hard to find the documentation for multiple authentication

**Comment by claudiodekker** [comment]
Hi,

Inertia by itself doesn't really provide any solution/approaches to authentication _or any other application specific feature_, as it's really only providing the single-page-app 'glue' between your (Vue/React/Svelte) views & back-end (Laravel, Ruby, etc..). 

If you want to create/implement either multi-factor (2FA) or multiple authentication methods (Google, Facebook, Twitter, etc..), there's nothing preventing you from doing so using any existing Vue/React/Svelte component libraries and back-end implementations that exist.

Good luck!
